src/parsers/FR/FRRelativeDateFormatParser.js

```diff
@@ -51,6 +51,11 @@ exports.Parser = function FRRelativeDateFormatParser(){
         var modifier = match[MODIFIER_1_GROUP] === undefined ?
                     (match[MODIFIER_2_GROUP] === undefined ? '' : match[MODIFIER_2_GROUP].toLowerCase())
                      : match[MODIFIER_1_GROUP].toLowerCase();
+        if(!modifier) {
+            // At least one modifier is mandatory to match this parser
+            result.tags['FRRelativeDateFormatParser'] = false;
```

Comment:
Lets move `result.tags['FRRelativeDateFormatParser'] = true;` further below just before returning the result instead?